qiskit_experiments/library/quantum_volume/qv_analysis.py

```diff
@@ -15,17 +15,123 @@
 
 import math
 import warnings
-from typing import Optional
+from typing import List
 
 import numpy as np
 import uncertainties
 from qiskit_experiments.exceptions import AnalysisError
-from qiskit_experiments.curve_analysis.visualization import plot_scatter, plot_errorbar
 from qiskit_experiments.framework import (
     BaseAnalysis,
     AnalysisResultData,
     Options,
 )
+from qiskit_experiments.visualization import BasePlotter, MplDrawer
+
+
+class QuantumVolumePlotter(BasePlotter):
+    """Plotter for QuantumVolumeAnalysis"""
```

Comment:
If this plotter isn't going to support more than one series, we need to make sure it's documented here so users are aware.


```suggestion
    """Plotter for QuantumVolumeAnalysis
    
    .. note::
    	This plotter only supports one series, the first of :attr:`series`. Additional series will be ignored.
    
    """
```